Makefile

```diff
@@ -130,7 +130,9 @@ pushtonexus:
 
 release_docu: docu_html docu_htmlnoheader
 	mkdir -p strimzi-$(RELEASE_VERSION)/docs
-	$(CP) -rv documentation/html/index.html strimzi-$(RELEASE_VERSION)/docs/
+	$(CP) -rv documentation/html/index.html strimzi-$(RELEASE_VERSION)/docs/using.html
```

Comment:
why index.html is copied as using.html?

Reply:
Because it seemed to be a bit confusing to me to have the one book there as index.html. It might confuse users to think that it includes the other books etc.

Reply:
After some more discussion I renamed the file back to index.html and will open a new issue to rename it everywhere.